contrib/recipes-devtools/gcc/gcc-7.3.inc

```diff
@@ -72,6 +72,7 @@ SRC_URI = "\
            file://0047-sync-gcc-stddef.h-with-musl.patch \
            file://0048-gcc-Enable-static-PIE.patch \
            file://0049-gcc-override-TARGET_LIBC_PROVIDES_SSP.patch \
+           file://0050-gcc-ustat-compiler-error-fix.patch \
```

Comment:
I'm fine with adding a patch for this, but I'd rather back-port the patch that went into gcc 7.4.0 upstream.

Reply:
Using the official 7.4.0 upstream patch would be better indeed.